### PR TITLE
Remove freenode mentions in remaining files

### LINKED ISSUE #1217: Replace kiwiirc irc links with https://web.libera.chat links
Hi @xarkes! I can work on this!

### PR DESCRIPTION
**Pull request by rbrishabh**
Fixes #1217

Hi @xarkes! How does this look?
let me know if this needs any changes!
Thanks! (PS: Sorry for copy-pasting the same message on all PRs -- I'm not the most innovative today!)

### CODE REVIEW

**Review by wargio — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -104,9 +104,9 @@ development, and general topics related to the project.
 
 We also provide the following partial bridges to other messaging platforms:
 - [Telegram](https://t.me/rizinorg)
-- IRC: irc.freenode.net
-  - Community - [#rizin](https://webchat.freenode.net/?channels=#rizin)
-  - Developers - [#rizindev](https://webchat.freenode.net/?channels=#rizindev)
+- IRC: web.libera.chat
```
Comment (XVilka) [inline README.md]: It's not. The right link for the IRC would be `irc.libera.chat`

Comment (XVilka) [reply README.md]: It has nothing to do with Web.

**Review by XVilka — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (wargio) [inline README.md]:
```suggestion
- IRC: [irc.libera.chat:6697 (TLS)](ircs://irc.libera.chat:6697)
```

File: README.md [hunk 2]
```diff
@@ -104,9 +104,9 @@ development, and general topics related to the project.
 
 We also provide the following partial bridges to other messaging platforms:
 - [Telegram](https://t.me/rizinorg)
-- IRC: irc.freenode.net
-  - Community - [#rizin](https://webchat.freenode.net/?channels=#rizin)
-  - Developers - [#rizindev](https://webchat.freenode.net/?channels=#rizindev)
+- IRC: [irc.libera.chat:6697 (TLS)](ircs://irc.libera.chat:6697)
```
Comment (wargio) [inline README.md]:
```suggestion
- IRC: [irc.libera.chat:6697 with TLS support](ircs://irc.libera.chat:6697)
```

**Pull request merged**